### ISSUE TITLE
[Usable – Make Code – Home page] There is no chevron or underline for footer link controls under ‘Home page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3514]
**User Experience:**
User will be misguided If any actionable control(link) doesn't have underline or chevron as they will skip the functionality of such link. Not only this, user will face disappointment if visually appearing link display as text as user won't be proceed further for relevant action.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify visual 'Microsoft Make Code, Terms of use, Privacy' link by using Developer tool(F12).

**Actual Result:**
There is no chevron or underline appears for visual 'Microsoft Make Code, Terms of use, Privacy' link controls under ‘Home page’ and due to this, user is getting affected as he/she might consider that link as text and will skip the functionality of such link.

**Expected Result:**
There must be chevron or underline on visual 'Microsoft Make Code, Terms of use, Privacy' link controls under ‘Home page’, so that user could easily navigate that link to get more details.

**Note:**
This issue also exists with Firefox and Chrome Browser.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#A11yUsable, #A11ySev4, #Accessibility, #Benchmark, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

![Usable_Link looks as a normal text](https://user-images.githubusercontent.com/72871089/97548401-18e0ce00-19f5-11eb-9684-59c11a902c16.jpg)

**Comment by abchatra** [comment]
This is same as microsoft.com homepage. This looks like a link.
